package.json

```diff
@@ -14,23 +20,35 @@
     "@wdio/local-runner": "^8.17.0",
     "@wdio/mocha-framework": "^8.17.0",
     "@wdio/spec-reporter": "^8.17.0",
+    "allure-mocha": "^2.9.2",
     "cross-env": "^7.0.3",
+    "mocha": "^10.2.0",
+    "mocha-multi-reporters": "^1.5.1",
     "ts-node": "^10.9.1",
     "typescript": "^5.2.2"
   },
   "scripts": {
     "test": "npm run build && wdio run ./src/config/wdio.conf.ts",
-    "test:ui": "npm run test --suite ui",
-    "test:run": "cross-env TEST_RUNNER=cucumber npm run test -- --cucumberOpts.tags='@run'",
+    "test:ui": "npm run test -- --suite ui",
+    "test:api": "npm run test -- --suite api",
```

Comment:
I suppose this script needs to be deleted

Reply:
No need to delete, framework allows to use wdio for api tests with specified allure realization